### ISSUE TITLE
CLS support config has old part names in it

### ISSUE THREAD

**Issue by Gordon-Dry** [steamp0rt/Kerbalism#40]
GameData\Kerbalism\Support\CLS.cfg:
```
@PART[Greenhouse]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}


@PART[GravityRing]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}
```

I guess it should be _kerbalism-greenhouse_ instead of _Greenhouse_
and _kerbalism-gravityring_ instread of _GravityRing_, or am I wrong?

**Comment by Gordon-Dry** [comment]
see
https://github.com/steamp0rt/Kerbalism/issues/39

**Comment by PiezPiedPy** [comment]
I'll check ;)

**Comment by PiezPiedPy** [comment]
this need closing, fixed with PR #42

**Comment by Gordon-Dry** [comment]
But I admit that on decoupling this still happens.
